composer.json

```diff
@@ -22,8 +22,8 @@
         }
     },
     "require": {
-        "php": ">=7.2",
```

Comment:
Using `>=` here was intentional see https://github.com/symfony/symfony/pull/36876. Since the downstream dependencies of this package (i.e. php-cs-fixer) use `^` version constraints it's unlikely that this package would ever be installed on a new version of PHP and break. Also this package is essentially just an array so I doubt it will ever break on a new version of PHP 😁. Using `^` for the version constraints will just make things more annoying for anyone trying to install this.

Reply:
Yeah, probably was too rushed when I changed it. It appears to work the way it is in fact. 👍